tasks/WebSocketTask.cpp

```diff
@@ -0,0 +1,141 @@
+/* Generated from orogen/lib/orogen/templates/tasks/Task.cpp */
+
+#include "WebSocketTask.hpp"
+
+#include <seasocks/PrintfLogger.h>
+#include <seasocks/WebSocket.h>
+#include <seasocks/Server.h>
+
+#include <json/json.h>
+
+#include <string.h>
+#include <iostream>
+
+using namespace controldev;
+using namespace seasocks;
+
+struct controldev::JoystickHandler : WebSocket::Handler {
+    WebSocket *connection = nullptr;
+    WebSocketTask *task = nullptr;
+    void onConnect(WebSocket *socket) override{
+        if (connection != nullptr){
+            connection->close();
+        }
+        Json::Value msg = true;
```

Comment:
Use a structured value, like `{ result: true }`, which will be easier to extend later if the need arises.